orchagent/portsorch.cpp

```diff
@@ -4159,7 +4174,19 @@ void PortsOrch::generatePriorityGroupMapPerPort(const Port& port)
 
         vector<FieldValueTuple> fieldValues;
         fieldValues.emplace_back(PG_COUNTER_ID_LIST, counters_stream.str());
+        m_flexCounterTable->set(key, fieldValues);
 
+        delimiter = "";
+        std::ostringstream ingress_pg_drop_packets_counters_stream;
+        key = getPriorityGroupDropPacketsFlexCounterTableKey(id);
+        /* Add dropped packets counters to flex_counter */
+        for (const auto& it: ingressPriorityGroupDroppedPktsStatIds)
+        {
+            ingress_pg_drop_packets_counters_stream << delimiter << sai_serialize_ingress_priority_group_stat(it);
+            delimiter = comma;
```

Comment:
@ayurkiv-nvda this is ugly way, even though it's commonly used in SONiC.
What about this one:
```
for (const auto& it: ingressPriorityGroupDroppedPktsStatIds)
{
    ingress_pg_drop_packets_counters_stream << delimiter << sai_serialize_ingress_priority_group_stat(it);
    if (delimiter.empty())
    {
        delimiter = comma;
    }
}
```

Reply:
Done